src/web/layouts/ShowcaseLayout.tsx

```diff
@@ -39,8 +40,9 @@ export const ShowcaseLayout = ({ CAPI, NAV }: Props) => {
 
     const adTargeting: AdTargeting = buildAdTargeting(CAPI.config);
 
-    // Currently hardcode this condition to true or false, so we can easily control the slot during development.
-    const renderBottomSlot = false;
+    // defaults to false, but use ?slot-machine-flags=showBodyEnd to show
+    const renderBottomSlot = parse(CAPI.slotMachineFlags || '').showBodyEnd;
```

Comment:
What if we rename this constant to `showBodyEnd` to match the flag name? Even better, we could rename the slot component itself as well so they all have a consistent name. I know this is likely to change soon but for now we could probably avoid having to translate `bottomSlot` to `bodyEnd` and `renderBottomSlot` to `showBodyEnd`. Happy to pick this up myself in a subsequent PR.

Perhaps consider destructuring afterwards e.g.
`const { showBodyEnd, someOtherFlagInTheFuture } = parse(CAPI.slotMachineFlags || '');`

Reply:
Hey sounds good - I'll add a commit!